### PR TITLE
graphics-drivers: fix SUMMARY and add index in index.md

### PR DESCRIPTION
**Pull request by ericonr**
Thanks @st3r4g for pointing it out c:

### CODE REVIEW

File: src/SUMMARY.md [hunk 1]
```diff
@@ -42,8 +42,9 @@
       - [ConnMan](./config/network/connman.md)
    - [Graphical Session](./config/graphical-session/index.md)
       - [Graphics Drivers](./config/graphical-session/graphics-drivers/index.md)
-         - [Intel GPU](./config/graphical-session/graphics/intel.md)
-         - [NVIDIA Optimus](./config/graphical-session/graphics/optimus.md)
+         - [Intel GPU](./config/graphical-session/graphics-drivers/intel.md)
+         - [NVIDIA
```
Comment (st3r4g) [inline src/SUMMARY.md:46]: extra newline?

Comment (ericonr) [reply src/SUMMARY.md:46]: `vmdfmt`, actually

Comment (st3r4g) [reply src/SUMMARY.md:46]: ah ok

**Comment by st3r4g** [comment]
@bobertlo can you merge this asap? Two pages are broken without this:
https://docs.voidlinux.org/config/graphical-session/graphics/intel.html
https://docs.voidlinux.org/config/graphical-session/graphics/optimus.html

**Pull request merged**